src/components/FloatingTabBar/FloatingTabBar.styles.tsx

```diff
@@ -0,0 +1,22 @@
+import { StyleSheet } from 'react-native';
+
+const backgroundColor = 'white';
+const margin = 20;
+const height = 70;
+
+// Content padding-bottom = 2 * margin + height
```

Comment:
@MauHylian con esto ocupamos calcular el padding del contenido

Reply:
Excelente @Canciller. Feel free to implement that.

Reply:
Nombre, es de lo tuyo